src/data/sponsors.js

```diff
@@ -15,4 +15,10 @@ export const sponsors = [
   //   websiteUrl: "https://example.com",
   //   tier: "Gold",
   // },
-];
\ No newline at end of file
+  {
+    name: "Flox",
+    logoUrl: "https://private-user-images.githubusercontent.com/65275785/451546602-1b774d2e-1358-42eb-9d19-4218814ff241.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDkwNjY5MDIsIm5iZiI6MTc0OTA2NjYwMiwicGF0aCI6Ii82NTI3NTc4NS80NTE1NDY2MDItMWI3NzRkMmUtMTM1OC00MmViLTlkMTktNDIxODgxNGZmMjQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjA0VDE5NTAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyNDY2OGJjYzhkY2UyYTA4YmYzNGIzNjdmZmRmYjYwMTU1YWRhNjc2NzhiNWU5MmY3ZmMxY2M1NmUzZDFlYjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.G1pWvScBVlPinuhCRQW_WTyv_S1iF0xFcTbiOXPqJuE",
```

Comment:
@das-g I guess you meant to post [your comment](https://github.com/nixcon/2025.nixcon.org/pull/40#issuecomment-2941353272) here

Indeed, this link is broken for people who don't have access to the internal repo.

Reply:
Ah. I hadn't payed attention to the time line and thought #40 was already an attempt at fixing the 404 error for the logo image.